src/components/gridList/demoDynamicTiles/index.html

```diff
@@ -1,18 +1,17 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
+  <md-button ng-click="vm.rotate()">Rotate</md-button>
```

Comment:
I think we either need to:
*  put this below the Grid or 
*  use a FAB in the TopLeft corner of the Grid.